standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/datasource/HikariCPDataSourceProvider.java

```diff
@@ -44,11 +44,11 @@ public DataSource create(Configuration hdpConfig) throws SQLException {
     LOG.debug("Creating Hikari connection pool for the MetaStore");
 
     String driverUrl = DataSourceProvider.getMetastoreJdbcDriverUrl(hdpConfig);
+    LOG.info("XXX got url " + driverUrl + " from DataSourceProvider");
```

Comment:
is this msg format intentional?

Reply:
XXX means I'm supposed to remove it before I commit, which obviously I failed to do.  I'll get rid of it.